Dockerfile

```diff
@@ -1,63 +1,16 @@
-# Galaxy - Exom Sequencing Pipeline
+# Galaxy - Exome Sequencing Pipeline
 
-FROM bgruening/galaxy-ngs-preprocessing
+FROM bgruening/galaxy-stable:16.04
```

Comment:
I think we should base this in ngs-preprocessing. We probably want to have the mapping tools and the QC in as well: https://github.com/bgruening/docker-galaxy-ngs-preprocessing/blob/master/ngs_preprocessing.yml

This means we do not need to install BWA and deeptools in this flavour.